src/storage/src/render/mod.rs

```diff
@@ -236,6 +236,10 @@ pub fn build_ingestion_dataflow<A: Allocate>(
     let worker_logging = timely_worker.log_register().get("timely");
     let debug_name = primary_source_id.to_string();
     let name = format!("Source dataflow: {debug_name}");
+    let cluster_size: Option<String> = timely_worker
+        .config()
+        .get("materialize/storage.cluster_size")
+        .map(|s: &String| s.clone());
```

Comment:
Made use of the timely config. Was there another way to get the cluster size value here?

Reply:
It seems better to make this part of `StorageState` instead of passing it through Timely.

Reply:
Let me look into that.

Reply:
Done!